### ISSUE TITLE
Help me with css configuration

### ISSUE THREAD

**Issue by giggioman00** [videojs/video.js#2364]
Hi. I'm here: http://designer.videojs.com/
and I want to make some changes... but there are 2 things that seems I cannot do.

1. I want to replace the diamond with a circle. Is it possible? How?
2. I want to change the buffer bar color. For the background of this bar, I want to use a pattern made with patternify just like the slider-bar-pattern. How can I do that? This is the bar I mean:
http://s8.postimg.org/nn4841v9h/image.jpg

Thank you

**Comment by mmcc** [comment]
1. We don't have a circle in the 4.x icon set, so you'll need to provide your own. If you're feeling gutsy, you could use a [unicode character](http://www.fileformat.info/info/unicode/char/25CF/index.htm) with wide support. That one specifically should work pretty well across platforms.
2. You'll need to base64 encode the new pattern you want to use and replace the old one in this line: https://github.com/videojs/video.js/blob/stable/src/css/video-js.less#L31. You would take your new base64 encoded image, and replace everything after `data:image/png;base64,` with it.

**Comment by giggioman00** [comment]
1. Thank you
2. Nope. That line change the pattern of the play bar (the one in striped blue). Please take a look at the image I have posted... I need to change color of the other bar (the buffer bar)... I don't find it anywhere

**Comment by mmcc** [comment]
Ah I misunderstood you. You can simply copy over the styles used for the [play progress class](https://github.com/videojs/video.js/blob/stable/src/css/video-js.less#L440-L452) for the [load progress class](https://github.com/videojs/video.js/blob/stable/src/css/video-js.less#L453-L456).

**Comment by giggioman00** [comment]
Thank you, it works. Now just another question: is it possible to use base 64 encode for customize the unicode circle?

**Comment by mmcc** [comment]
Sure. FWIW, there's a circle in the 5.0 icon set.

**Comment by giggioman00** [comment]
I don't understand. in http://designer.videojs.com/ I've done my work, the player have a the style I want. So I saved the LESS and css part on my desktop. I renamed the first in video-js.less and the second in video-js.css. 

But I have problem with the control bar. For example, control bar have the same color I have set on the file. But is in the middle of the video, and doesn't appears the play bar and the buffer bar. I don't understand, in that site everything works well. What's the problem?
